### PR TITLE
Change default project for Event Adapter

### PR DESCRIPTION
**Pull request by erocchi**

### CODE REVIEW

File: event-adapter/Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 OUT_DIR = build
 PACKAGE = k8s.io/event-adapter
-PREFIX = gcr.io/erocchi-gke-dev-1attempt
+PREFIX = gcr.io
+GCP_PROJECT ?= google-containers
```
Comment (kawych) [inline event-adapter/Makefile]: I'd prefer `PREFIX ?= gcr.io/google-containers`

**Comment by kawych** [comment]
Please update the description to say which project you contribute to, e.g. "Change default project for Event Adapter"

File: event-adapter/Makefile (hunk 1 above)
Comment (erocchi) [reply event-adapter/Makefile]: Done

**Pull request merged**